python/mleap/sklearn/svm.py

```diff
@@ -38,11 +38,6 @@ def deserialize_from_bundle(self, path, node_name):
     serializer = SimpleSerializer()
     return serializer.deserialize_from_bundle(self, path, node_name)
 
-setattr(SVC, 'op', 'svm')
```

Comment:
MLeap only supports a linear kernel, so only the `kernel='linear'` option is valid for this class. Because of this, I tried to make the serializer error any time a different `kernel` option was chosen. But even with a linear kernel, the `coef_` matrix of this class is a read-only derived attribute:

https://stackoverflow.com/questions/34827838/scikit-learn-numpy-changing-the-value-of-an-read-only-variable

So we would have to reconstruct the coefficients from `model.dual_coef_` and `model.support_vectors_` when deserializing. Rather than doing this, I thought we should just remove `SVC` support all-together, as the `kernel='linear'` case is already covered by `LinearSVC` anyway

Reply:
Also... there were no tests for this class. And if you try to actually use MLeap to serialize `SVC` it fails . So it's safe to say nobody is using this feature